api/server.py

```diff
@@ -296,6 +301,42 @@ def add_project():
     project_id = projects.insert(project)
     return str(project_id)
 
+## testing description scraping
+@app.route('/api/projects/descrp', methods=['POST'])
```

Comment:
Let's use `/api/projects/scrape-project-descriptions-from-devpost` (to be consistent with Naveen's PR)